diff --git a/lib/Drupal/Core/Security/DoTrustedCallbackTrait.php b/lib/Drupal/Core/Security/DoTrustedCallbackTrait.php
index 4548a0a52b..de53cef417 100644
--- a/lib/Drupal/Core/Security/DoTrustedCallbackTrait.php
+++ b/lib/Drupal/Core/Security/DoTrustedCallbackTrait.php
@@ -48,7 +48,7 @@ trait DoTrustedCallbackTrait {
    *
    * @see \Drupal\Core\Security\TrustedCallbackInterface
    */
-  public function doTrustedCallback(callable $callback, array $args, $message, $error_type = TrustedCallbackInterface::THROW_EXCEPTION, $extra_trusted_interface = NULL) {
+  public function doTrustedCallback(callable|array $callback, array $args, $message, $error_type = TrustedCallbackInterface::THROW_EXCEPTION, $extra_trusted_interface = NULL) {
     $object_or_classname = $callback;
     $safe_callback = FALSE;
 
@@ -100,4 +100,4 @@ trait DoTrustedCallbackTrait {
     return call_user_func_array($callback, $args);
   }
 
-}
+}
\ No newline at end of file
