diff --git a/src/Plugin/Field/FieldWidget/EntityReferenceBrowserWidget.php b/src/Plugin/Field/FieldWidget/EntityReferenceBrowserWidget.php
index 42c9da7..21e9642 100644
--- a/src/Plugin/Field/FieldWidget/EntityReferenceBrowserWidget.php
+++ b/src/Plugin/Field/FieldWidget/EntityReferenceBrowserWidget.php
@@ -459,6 +459,9 @@ class EntityReferenceBrowserWidget extends WidgetBase implements ContainerFactor
    * Render API callback: Processes the entity browser element.
    */
   public static function processEntityBrowser(&$element, FormStateInterface $form_state, &$complete_form) {
+    if (empty($element['#attached']['drupalSettings']['entity_browser'])) {
+      return $element;
+    }
     $uuid = key($element['#attached']['drupalSettings']['entity_browser']);
     $element['#attached']['drupalSettings']['entity_browser'][$uuid]['selector'] = '#' . $element['#custom_hidden_id'];
     return $element;
